CourseParsing/ParseScript.py

```diff
@@ -8,7 +8,7 @@
 def get_course_codes():
     dbc = DatabaseReceiver()
     course_code_query = dbc.select(what="course_codes", table="requirements")
-    course_codes = set([])
+    course_codes = []
```

Comment:
Is there a reason this is changed from a set to an array? From what I remembered the requirements table has reqs from many years - which has duplicate course codes (for ex MATH 135 in many programs/years) and I think that was why a set was used. 

Have you tested this out?

Reply:
Line 50 reverses the course_codes, which is not possible for a set. So we converted it to a list, unless you wanted different functionality.

Reply:
Sounds good

Reply:
I just wanted to make sure no duplicated courses are displayed